.prettierrc.js

```diff
@@ -1,7 +0,0 @@
-module.exports = {
```

Comment:
Why you deleted it?

Reply:
Because it's included in the `.eslintrc` file, in section `"rules" -> "prettier/prettier"`. 
The idea was to follow the same setup which is on the web

Reply:
printWidth 150 is too big. You even can check how this code got ugly. Let's setup 100
Probably web developers set 150 because they use css like stuff.

Reply:
I think would be better to discuss this with the team, since decision about 150 was not decided by me :)